app/js/services/situationService.js

```diff
@@ -197,6 +197,32 @@ angular.module('ddsApp').factory('SituationService', function($http, $sessionSto
 
         hasEnfant: function(situation) {
             return _.find(situation.individus, { role: 'enfant' });
+        },
+
+        hasConjoint: function(situation) {
+            return _.find(situation.individus, { role: 'conjoint' });
+        },
+
+        setConjoint: function(situation, conjoint) {
+            var individus = situation.individus;
+            // si le conjoint existait déjà avant, on l'écrase
+            if (this.hasConjoint(situation)) {
+                individus[individus.length - 1] = conjoint;
+            } else {
+                // on insère le conjoint en dernier dans la liste des individus
+                individus.push(conjoint);
+            }
+        },
+
+        setEnfants: function(situation, enfants) {
+            var individus = situation.individus;
+            individus = _.filter(individus, function(individu) {
+                return 'enfant' !== individu.role;
+            });
+            individus = individus.slice(0,1)
+                .concat(enfants)
+                .concat(individus.slice(1));
```

Comment:
With splice, the result would be : `[demandeur, [enfant1, enfant2], conjoint]` while we want `[demandeur, enfant1, enfant2, conjoint]`.
We could use `individus.splice.apply(...) but that would be less readable`